app/templates/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "scripts": {
+    "start": "node index.js"
+  },
+  "dependencies": {
+    "h5bp": "^0.1.3"
+  }
+}
```

Comment:
Let's don't call this file `package.json` as NPM will try to use it and fail. Just call it `_package.json` and copy it to `package.json`.